components/comment/index.en-US.md

```diff
@@ -0,0 +1,30 @@
+---
+category: Components
+type: Data Display
+title: Comment
+cols: 1
+---
+
+Single comment component.
+
+## When To Use
+
+Comments can be used to enable discussions on an entity for example page, blog post, issue or other.
+
+## API
+
+| Property | Description | Type | Default |
+| -------- | ----------- | ---- | ------- |
+| actions | List of action items rendered below the comment content | Array<ReactNode> | - |
+| author | The element to display as the comment author | ReactNode | - |
+| avatar | The element to display as the comment avatar - generally an antd `Avatar` | ReactNode | - |
+| children | Nested comments should be provided as children of the Comment | ReactNode | - |
+| className | Addtional className for comment | string | - |
+| content | The main content of the comment | ReactNode | - |
+| contentStyle | Inline style to apply to the comment content | object | - |
+| avatarStyle | Inline style to apply to the comment avatar | object | - |
+| id | Optional ID for the comment | string | - |
+| innerStyle | Additional style for the inner content | object | - |
+| prefixCls | Comment prefix className defaults to `.ant-comment` | string | .ant-comment |
```

Comment:
We don't expose `prefixCls` as offcial prop cause we don't want people change that, like other components did.